BaikalStandalone/Application/cl_render.cpp

```diff
@@ -71,7 +71,10 @@ namespace Baikal
 
         for (std::size_t i = 0; i < m_cfgs.size(); ++i)
         {
-            std::cout << i << ": " << m_cfgs[i].context.GetDevice(0).GetName() << "\n";
+            std::cout << i << ". name: " << m_cfgs[i].context.GetDevice(0).GetName()
```

Comment:
Make some variable. For example  some_variable  = m_cfgs[i].context.GetDevice(0).